include/RED4ext/RedProcess.hpp

```diff
@@ -0,0 +1,78 @@
+#include <RED4ext/RED4ext.hpp>
+
+namespace RED4ext::red
+{
+
+struct Process
+{
+    struct FixedWString
+    {
+        uint32_t length;
+        uint32_t maxLength;
+        const wchar_t* str;
+    };
+
+    enum class ExecutionFlags : unsigned char
+    {
+        // unsets CREATE_NO_WINDOW
+        ShouldCreateWindow = 0x1,
+        // sets CREATE_BREAKAWAY_FROM_JOB | CREATE_SUSPENDED in Process Creation Flags
+        BreakawayAndSuspend = 0x2,
+        // unsets bInheritHandles
+        NoInheritHandles = 0x4
+    };
+
+    enum class ReadFlags : unsigned char
+    {
+        // open & read regardless of other options
+        Unk1 = 0x1,
+        // opens file from name provided and writes output
+        WriteToFile = 0x2,
+        // writes output to CString
+        GetCString = 0x4
+    };
+
+    static const constexpr uint32_t defaultTimeout = 60000;
+
+    // creates the read & write pipes, and handle information
+    /// @pattern 48 89 5C 24 08 57 48 83 EC 40 33 C0 C7 44 24 20 18 00 00 00 48 8D 91 08 20 00 00 48 89 81 00 20
```

Comment:
Should we keep these comments here?

Reply:
I think so - the functions aren't very useful down the road if we don't know where they came from :)